__tests__/app.test.js

```diff
@@ -0,0 +1,121 @@
+const { app } = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test("GET 200: Responds with an array of topic objects with slugs and descriptions", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const { topics } = response.body;
+        expect(topics).toEqual([
+          {
+            description: "The man, the Mitch, the legend",
+            slug: "mitch",
+          },
+          {
+            description: "Not dogs",
+            slug: "cats",
+          },
+          {
+            description: "what books are made of",
+            slug: "paper",
+          },
+        ]);
+      });
+  });
+  test("400: responds with correct error code and message when invalid method is used", () => {
+    return request(app)
```

Comment:
We don't require these tests. The task is only concerning the GET for /api/topics. I can see what you're trying to do here, but we're only concerned about the specific method with these tests. We'll be looking at a specific way to handle this sort of thing in tomorrow's lecture, but for now all of these tests can be removed 👍